oojs.js

```diff
@@ -10,3 +10,21 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+
+class Playlist{
+  constructor(title, songs){
+    this.title = title
+    this.songs = songs
+    this.favorites = 0
+  }
+  addSong(song){
+    this.songs.push(song);
+  }
+}
+
+class Album extends Playlist{
+  constructor(){
+    this.artist = artist
```

Comment:
the constructor would have to take 3 arguments, `title` and `songs` to pass to the parent's construct via `super(title, songs)` in `Album`'s constructor. super must be called in the **inheriting** class' constructor function.

Everything else looks solid!